AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/Resolvers/Authentication/AuthenticationState+Resolver.swift

```diff
@@ -19,15 +19,23 @@ extension AuthenticationState {
         ) -> StateResolution<StateType> {
             switch oldState {
             case .notConfigured:
-                guard let authEvent = event as? AuthenticationEvent else {
+                if let authEvent = event as? AuthenticationEvent {
+                    return resolveNotConfigured(byApplying: authEvent)
+                } else if let authZEvent = event.isAuthorizationEvent,
```

Comment:
Should we accept event before the authentication is configured?

Reply:
Yep.. Because federation could still be possible if UP is not configured.